content/applications/finance/accounting/fiscal_localizations/localizations/united_kingdom.rst

```diff
@@ -0,0 +1,84 @@
+==============
+United Kingdom
+==============
+
+Configuration
+=============
+
+:ref:`Install <general/install>` the following modules to get all the features of the UK
+localization:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Technical name
+     - Description
+   * - :guilabel:`UK - Accounting`
+     - `l10n_uk`
+     - Installing this module grants you access to:
+        - CT600-ready chart of accounts
+        - VAT100-ready tax structure
+        - InfoLogic UK counties listing
+        - a few other adaptations
+   * - :guilabel:`UK - Accounting Reports`
+     - `l10n_uk_reports`
+     - Installing this module grants you access to:
+        - Accounting reports for UK
+        - Allows sending the tax report via the MTD-VAT API to HMRC.
+
+.. tip::
+   Installing the module :guilabel:`UK - Accounting Reports` installs all two modules at
+   once.
+
+.. note::
+   Your company should be located in the UK to be able to submit HMRC reports.
+
+.. seealso::
+   - `HM Revenue & Customs <https://www.gov.uk/government/organisations/hm-revenue-customs/>`_
+   - `Overview of Making Tax Digital <https://www.gov.uk/government/publications/making-tax-digital/overview-of-making-tax-digital/>`_
+
+Making Tax Digital (MTD)
+========================
+
+Making Tax Digital is a key part of the UK government’s plans to make it easier for individuals and
+businesses to get their tax right and keep on top of their affairs. All VAT-registered businesses
+are required to follow the MTD rules by keeping digital records and using software to submit their
+VAT returns.
+
+The **UK - Accounting Reports** package enables you to comply with the `HM Revenue & Customs <https://www.gov.uk/government/organisations/hm-revenue-customs/>`_
+requirements regarding `Making Tax Digital <https://www.gov.uk/government/publications/making-tax-digital/overview-of-making-tax-digital/>`_ in the UK.
+
+.. important::
```

Comment:
```suggestion
.. important::
```
I'll let you decide, but this part *seems to me* about VAT return submission, which is nowhere to be found in the doc. Access as well as registering to HMRC are explained, but not how to submit. If I remember correctly, it is explained in the video "Tax report" with Eva, otherwise you can always find the script in my deployed tasks

Reply:
This is now part of the Taxes section.
The submission part is already explained in the next section.
I added a see also to the tutorial, good idea. However